### PR TITLE
Update the dojo example to use the new UI and npm.

### PR DESCRIPTION
**Pull request by edchat**
https://github.com/tastejs/todomvc/issues/1110

### CODE REVIEW

File: examples/dojo/package.json [hunk 1]
```diff
@@ -1,4 +1,13 @@
 {
+	"private": true,
+	"dependencies": {
+		"todomvc-app-css": "^1.0.0",
```
Comment (arthurvr) [inline examples/dojo/package.json]: package.json uses and will always use 2 spaces. [That's stated by our editorconfig too](https://github.com/tastejs/todomvc/commit/64484b9e18eabed01b8314af899f50697d167946).

File: examples/dojo/.gitignore [hunk 2]
```diff
@@ -1,8 +1,13 @@
...
 !js/lib/dojo
 js/lib/dojo/*
 !js/lib/dojo/dojo.js
+
+node_modules/*
+
+!node_modules/todomvc-app-css/index.css
+
+!node_modules/todomvc-common/base.css
+!node_modules/todomvc-common/base.js
```
Comment (arthurvr) [inline examples/dojo/.gitignore]: You'll have to actually include these too.

Comment (edchat) [inline examples/dojo/.gitignore]: Ok, I will update those. thanks.

**Comment by sindresorhus** [comment]
Thank you :dancer:

File: examples/dojo/package.json [hunk 3]
```diff
@@ -1,8 +1,17 @@
...
-	}
-}
+  "private": true,
+  "dependencies": {
+    "todomvc-app-css": "^1.0.0",
+    "todomvc-common": "^1.0.1",
+    "dojo": "1.10.2",
+    "dijit": "git://github.com/dojo/dijit.git#1.10.2",
+    "dojox": "git://github.com/dojo/dojox.git#1.10.2",
+    "util": "git://github.com/dojo/util.git#1.10.2"
```
Comment (sindresorhus) [inline examples/dojo/package.json]: Can you try to get these on npm? (like opening issues on the relevant repos about it)

**Pull request closed without merging**